### DISCUSSION TITLE
Error when using Ultimate SD Upscale + Controlnet

### DISCUSSION THREAD

**Discussion by HostileSausage** [vladmandic/sdnext#1156] [Q&A]
`Given groups=1, weight of size [64, 3, 3, 3], expected input[1, 4, 192, 192] to have 3 channels, but got 4 channels instead
`

Trying to upscale a 1024 x 1536 image using Ulti SD Upscale at 2x

ControlNet is Tile Resample with the `control_v11f1e_sd15_tile` model

Not sure if it's a setting or what but these exact same settings work fine in Auto1111, so I'm not sure what's causing it

**Comment by iDeNoh** [comment]
Check and see if you have anything set for "Default upscaler for image resize operations" in upscaler settings, of so change it to none and see if that fixes it.

**Reply by HostileSausage** [reply]
Yep that solved it I can upscale now in img2img.

Also one more thing, when using hiresfix in txt2img, I run out of memory when the batch size is more than 3, yet when I use regular A1111, I can set the batch size to 6 and I don't run out of memory. Is there a setting somewhere that might be causing this?

**Reply by iDeNoh** [reply]
That could be a lot of things, what platform are you on/GPU do you have?

**Comment by iDeNoh** [comment]
Also, as an aside, I would recommend using multidiffusion with controlnwt tile, you can get *way* better upscaled with them

**Reply by HostileSausage** [reply]
I'm on a 4090 and Windows 11.

Thanks I've never heard of multidiffusion I'll look into it.
